packages/rlc-common/src/buildObjectTypes.ts

```diff
@@ -54,6 +75,66 @@ export function buildObjectInterfaces(
   return objectInterfaces;
 }
 
+const MULTIPART_FILE_METADATA_PROPERTIES: OptionalKind<PropertySignatureStructure>[] =
+  [
+    {
+      name: "filename",
+      hasQuestionToken: true,
+      type: "string"
+    },
+    {
+      name: "contentType",
+      hasQuestionToken: true,
+      type: "string"
+    }
+  ];
+
+function buildMultipartPartDefinitions(
```

Comment:
Do we consider the case for  `A extends B`?

Reply:
Not currently but we can revisit if this becomes a requirement